builtInFunctions/multiESDTNFTTransfer.go

```diff
@@ -432,7 +432,12 @@ func (e *esdtNFTMultiTransfer) transferOneTokenOnSenderShard(
 			return nil, err
 		}
 	} else {
-		err = e.esdtStorageHandler.AddToLiquiditySystemAcc(esdtTokenKey, esdtData.Type, transferData.ESDTTokenNonce, big.NewInt(0).Neg(transferData.ESDTValue))
+		hasDynamicRole, err := hasDynamicRole(acntSnd, esdtTokenKey, e.marshaller)
```

Comment:
to improve the understanding of the code I would have named the `hasDynamicRole` to `keepMetadataOnZeroLiquidity`
Reason: we all know how the variable is populated (by calling the function hasDynamicRole) but is more obscure to understand what is the parameter of the AddToLiquiditySystemAcc function

Reply:
Good idea, renamed.